src/styles/cards.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+
+  height: 408px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 5px;
+  background-color: $primary-bg-color;
+  border: 1px solid $primary-bg-color;
+
+  &__image {
+    margin: 0 auto;
+    padding: 32px 19px 0;
+    height: 134px;
+    width: 160px;
+
+    object-fit: contain;
+    object-position: center;
+  }
+
+  &__product-name {
+    margin: 0;
+    padding: 40px 16px 4px;
+    height: 36px;
+
+    text-align: start;
+    line-height: 18px;
+    font-weight: 500px;
+    font-size: 12px;
+    color: $main-text-color;
+  }
+
+  &__product-code {
+    height: 14px;
+    margin: 0 16px;
+    display: flex;
+    justify-content: start;
+
+    color: $secondary-text-color;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+  }
+
+  &__review {
+    height: 16px;
+    margin: 16px 16px 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__review-count {
+    margin-top: 2px;
+
+    text-align: center;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    height: 18px;
+    margin: 24px 16px 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-text {
+    color: $secondary-text-color;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-tag {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+  &__button {
+    height: 40px;
+    width: 166px;
+    margin: 16px;
+
+    font-size: 14px;
+    line-height: 16px;
+    color: $primary-bg-color;
+    text-transform: uppercase;
+    font-weight: 700;
+
+    border: none;
+    border-radius: 5px;
+    background-color: $button-main-color;
+    transition: color 150ms ease;
```

Comment:
150ms feels a little to fast for the transition, while using the page you can barely notice it, maybe try 300ms or more 
**This is a personal suggestion this change is not required**